### ISSUE TITLE
Unlabeled Checkbox in the Navigation Menu

### ISSUE THREAD

**Issue by austinseraphin** [jekyll/minima#578]
The navigation menu has an unlabeled checkbox which appears to do nothing from a screen reader's perspective. I would suggest labeling it. I notice that earlier versions had an unlabeled clickable element, perhaps you transitioned to the checkbox. If I knew its function I might suggest a more accessible alternative, but start with labeling it and let's see how it goes.

**Comment by jekyllbot** [comment]
This issue has been automatically marked as stale because it has not been commented on for at least two months.

The resources of the Jekyll team are limited, and so we are asking for your help.

If this is a **bug** and you can still reproduce this error on the <code>master</code> branch, please reply with all of the information you have about it in order to keep the issue open.

If this is a feature request, please consider whether it can be accomplished in another way. If it cannot, please elaborate on why it is core to this project and why you feel more than 80% of users would find this beneficial.

This issue will automatically be closed in two months if no further activity occurs. Thank you for all your contributions.

**Comment by austinseraphin** [comment]
The issue remains.
